src/scripts/updateRequests/index.ts

```diff
@@ -0,0 +1,52 @@
+import { MongoClient } from 'mongodb';
+
+// Connection URL and Database Name
+const url = process.env.DATABASE_URL;
+const dbName = 'test-new-schema';
```

Comment:
I think it would be better for the data base name to be retrieved from an environment variable. But I don't think we have one for the DB name so for the purpose of this script I hard coded it.